### PR TITLE
Added GameFixes with elytra bug blocking

### PR DESCRIPTION
**Pull request by TealNerd**

### CODE REVIEW

**Comment by TealNerd** [comment]
@ProgrammerDan

File: src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/GameFixes.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+		StringBuilder genStatus = new StringBuilder();
+		genStatus.append("GameFixes is ");
+		if(config != null && config.isEnabled()) {
+			genStatus.append("active\n");
+			if(config.isBlockElytraBreakBug()) {
+				genStatus.append("   Block eltra break bug is enabled\n");
+				genStatus.append("   Will deal " + config.getDamageOnElytraBreakBug() + " damage to players\n");
+			} else {
+				genStatus.append("   Block elytra break bug is disabled\n");
+			}
```
Comment (ProgrammerDan) [inline src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/GameFixes.java:69]: Possible to add the other fixes to status here?

**Comment by ProgrammerDan** [comment]
Thanks for adding this btw, these are great.

File: src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/GameFixes.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+		}
+		return genStatus.toString();
+	}
+	
+	@EventHandler(priority=EventPriority.LOWEST, ignoreCancelled=true)
+	public void onBlockBreak(BlockBreakEvent event) {
+		Block block = event.getBlock();
+		Player player = event.getPlayer();
+		
+		if(!player.getLocation().equals(block.getLocation())
```
Comment (ProgrammerDan) [inline src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/GameFixes.java:81]: Need to check if actually in elytra mode, as this is also cancelling things like
"Player Head is in Stone Button" while mining normally.

**Pull request merged**